Src/Support/Google.Apis/Upload/ResumableUpload.cs

```diff
@@ -711,7 +706,7 @@ private void MediaCompleted(HttpResponseMessage response)
         // TODO: Make this and the next method read the stream using ReadAsync?
 
         /// <summary>Prepares the given request with the next chunk in case the steam length is unknown.</summary>
-        private void PrepareNextChunkUnknownSize(Stream stream, CancellationToken cancellationToken, out byte[] chunk, out int chunkLength)
+        private async Task<Tuple<byte[], int>> PrepareNextChunkUnknownSizeAsync(Stream stream, CancellationToken cancellationToken)
```

Comment:
A ValueTuple would avoid the extra allocation

Reply:
Yes but it does require an extra NuGet package which I wanted to avoid. This way the changes are absolutely minimal.

Reply:
Yes, although a value tuple would be better, we generally don't wish to add extra dependencies. So using `Tuple` is fine.